pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -324,6 +348,14 @@ func (c *APIClient) connect() error {
 			log.Infof("Could not get informer factory: %v", err)
 			return err
 		}
+		if c.CRDInformerFactory, err = getCRDInformerFactory(); err != nil {
+			_ = log.Errorf("Error getting datadoghq Client: %s", err.Error())
```

Comment:
we should probably have dedicated log lines for the two with an explicit client nmae

Reply:
yes I can do that change

Reply:
Done! f79a111442